src/renderSwitch.tsx

```diff
@@ -34,15 +38,27 @@ export async function renderSwitch(
     ...options.config,
   }
 
-  if (isTemplate(block)) {
-    html = await renderTemplate(block, {
-      basePath,
-      locals: {
-        ...locals,
-        ...data,
-        ...templates,
-      },
-    })
+  if (isString(block)) {
+    if (HTML_TAG.test(block)) {
+      html = block
```

Comment:
This is the important part of this PR for the work done to fix https://artsyproduct.atlassian.net/browse/PLATFORM-1184.

Reaction now is entirely in charge of rendering HTML, which it then just passes off to Force for inclusion as body, which means Stitch needs to accept already rendered HTML.

Reply:
👍

Reply:
I wonder if we might be able to test for an actual file down below so that the passed in string doesn't have to be tested against a regex. Like 

```js
if (isString(block) {
  const filePath = path.join(basePath, block)
  if (fs.stat(filePath) {
    ... 
  } else {
    html = block
  }
}
```

(or something)

Reply:
I was thinking of that first too, but a FS stat felt more expensive and would probably need to switch this codepath to async, so I didn’t go with that yet. I figured we’d give some more thought to a solution that wouldn’t require any inference on Stitch’s part.